src/compat/pg_compat.c

```diff
@@ -48,7 +48,12 @@ create_plain_partial_paths(PlannerInfo *root, RelOptInfo *rel)
 {
 	int			parallel_workers;
 
+#if PG_VERSION_NUM >= 110000
+	parallel_workers = compute_parallel_worker(rel, rel->pages, -1,
```

Comment:
Could you extract this one as `compute_parallel_worker` macro, please?

Reply:
done